xmtp_mls/src/identity_updates.rs

```diff
@@ -75,6 +78,33 @@ where
 
         Ok(get_state(updates)?)
     }
+
+    pub fn get_association_state_diff<InboxId: AsRef<str>>(
+        &self,
+        conn: &'a DbConnection<'a>,
+        inbox_id: String,
+        starting_sequence_id: Option<i64>,
+        ending_sequence_id: Option<i64>,
```

Comment:
The latest iteration of the protos don't use sequence_id, but a list of old and new updates: https://github.com/xmtp/proto/pull/160/files

Not using sequence ID means that the validation service can work with `IdentityUpdate` instead of `IdentityUpdateLog`, and it means validation can be done on the client and/or before insertion into the server database. Your call here!

Reply:
This is for the client's need to generate a diff when evaluating commits. There we need to search the SQLite database for ranges of `sequence_id`s before we can get the Identity Updates